src/Explorer/Explorer.tsx

```diff
@@ -924,7 +993,26 @@ export default class Explorer {
     await this.notebookManager?.notebookContentClient.updateItemChildrenInPlace(item);
   }
 
-  public openNotebookTerminal(kind: ViewModels.TerminalKind): void {
+  public async openNotebookTerminal(kind: ViewModels.TerminalKind): Promise<void> {
+    if (userContext.features.phoenix === true) {
```

Comment:
>erContext.features.phoenix === true [](http://example.com/codeflow?start=10&length=35)

we should util method instead of doing this variable check in multiple places